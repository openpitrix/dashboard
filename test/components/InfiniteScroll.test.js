import InfiniteScroll from 'components/InfiniteScroll';

describe('Component InfiniteScroll', () => {
  it('basic render', () => {
    const wrapper = render(<InfiniteScroll isLoading>test</InfiniteScroll>);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
