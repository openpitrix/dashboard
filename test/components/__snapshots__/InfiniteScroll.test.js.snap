// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component InfiniteScroll basic render 1`] = `
<div
  class=""
>
  test
</div>
`;
